hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/client/PortableReadResultSet.java

```diff
@@ -36,12 +35,14 @@
 import static com.hazelcast.ringbuffer.impl.client.RingbufferPortableHook.READ_RESULT_SET;
 import static java.util.Collections.unmodifiableList;
 
+// This class is not used in serialized form since at least 3.9, however is still
+// maintained as Portable for compatibility
```

Comment:
So, by 3.11 we can remove `Portable` completely?

Reply:
My main concern here is about compatibility with older clients; in order to safely remove `Portable` in 3.x series we need to check that no client since 3.6 uses this class in serialized form.